### PR TITLE
Added install message banner

### PR DESCRIPTION
**Pull request by ntotten**
image

### CODE REVIEW

File: workbench/header.php [hunk 1]
```diff
@@ -50,6 +50,11 @@
               "Workbench is currently undergoing maintenance. The service may be intermittently unavailable during this time.</div><br/>";
 }
 
+if (WorkbenchConfig::get()->isConfigured("displayInstallWarningMessage")) {
```
Comment (vazexqi) [inline workbench/header.php]: I will check to see if this setting is on in the production app.

Comment (vazexqi) [reply workbench/header.php]: @ntotten - I don't see `displayInstallWarningMessage` in the code base (or in https://github.com/forceworkbench/forceworkbench/blob/master/workbench/config/defaults.php).

Where did you get this config from?

Comment (vazexqi) [reply workbench/header.php]: It must be working somehow since it shows up for you but I'm just curious where that config is.

Comment (ntotten) [reply workbench/header.php]: PHP! It's all dynamic. I didn't add it yet to production.

Comment (vazexqi) [reply workbench/header.php]: It's not showing up in the review app as well - https://workbench-ps-staging-pr-823.herokuapp.com/login.php

**Pull request merged**